java/1.2/modules/ROOT/pages/configure-java-module-acb.adoc

```diff
@@ -0,0 +1,306 @@
++
+. Change the name of the flow: 
+.. Click *Flow*. 
+.. Click the edit icon. 
++
+image:connectors::acb-change-flow-name.png["Arrow pointing to the flow name edit pencil icon"]
++
+.. Enter the flow name, and click the checkmark. 
+
+[[configure-trigger]]
+== Configure a Trigger for the Flow
+A trigger (source) starts the flow when specified conditions are met. If a connector has its own associated sources, you can use a connector-specific trigger to initiate the flow. Each connector trigger has a unique configuration.
+
+You can configure the following sources for this connector:
+
+include::connectors::partial$t-studio-partial.adoc[tags=listener-scheduler-descriptions]
+
+[[adding-components-to-project]]
+== Adding Components to Your Project
+
+include::connectors::partial$configure-acb-connectors-partial.adoc[tag="acb-add-components-to-project"]
+
+[[configuring-component-attributes]]
+== Configuring the Component Attributes
+
+include::connectors::partial$configure-acb-connectors-partial.adoc[tag="acb-configure-attributes"]
+
+[[configuring-new]]
+=== Configuring the New Operation
+
+Create a new instance of a class by using the *New* operation of the Java module. The identifier of a constructor includes the class and constructor names, and the constructor has a full description of its signature including the types of each parameter.
+
+For example, to invoke the constructor `Me(String name, int age)` which belongs to the `org.bar.Me` class, then the identifier of the method is `Me(String,int)`.
+
+To configure the attributes for *New* in the Anypoint Code Builder UI:
+
+. Click the *New* component on the canvas.
+. In the *General* tab:
+.. Select the *Class* that contains the method to use.
+.. Select the *Method* to use. 
+. Select the *Advanced* tab to configure the following attributes: 
++
+[%header%autowidth.spread]
+|===
+| UI Field | Description | Required
+| Target variable | Name of the variable to assign the instance to. | No
+| Target value | Expression that evaluates the operation's output. The expression outcome is stored in the target variable. | No
+| Error mappings | This attribute is not currently enabled in the UI. Set of error mappings. | No
+|===
+. Select the *Input/Output* tab to filter and view input and output data for the *Invoke* operation.
+
+After you add your instance, <<configuring-invoke,configure the *Invoke* operation>>.
+
+[[configuring-invoke]]
+=== Configuring the Invoke Operation
+
+Invokes methods with the provided arguments in an instance. The identifier of the method to invoke includes the class and method names.
+
+To configure the attributes for *Invoke* in the Anypoint Code Builder UI:
+
+. Click the *Invoke* component on the canvas.
+. In the *General* tab:
+.. Select the *Instance* on which the method is invoked.
+.. Select the *Class* that contains the method to use.
+.. Select the *Method* to use. 
```

Comment:
Do you think a note below "Select the *Method* to use" (like the one you added below) could be useful here, as well? If this is a first-time config scenario, the issue won't occur, but if they're using for reference and change the class, it could. 

> NOTE: If this is blank, click the refresh icon to populate it.